### PR TITLE
Feature/item class

### PR DESCRIPTION
**Pull request by daudi13**
In this PR we:
- [ ] created class Item in item.rb file
- [ ] Initialized  class Item
- [ ] Implemented methods:
   - can_be_archived in the item class
   - move_to_archive in the item class

### CODE REVIEW

**Review by Tufahel — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 

The project is complete! Great work! There is nothing else to say other than... it's time to merge it :+1:

File: item.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Item
+  attr_accessor :publish_date
+  attr_reader :archived, :id, :genre, :label, :author
+
+  def initialize(publish_date, archived: false, id: nill)
```
Comment (iLynette) [inline item.rb:5]: Hello, @daudi13 this is okay but when data is saved, it has an id so you can set it as optional but have it as `id: rand(1..10_000)`

File: item.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    super()
+    @id = id || Random.rand(1..1000)
+    @archived = archived
+    @publish_date = publish_date
+    @label = nill
```
Comment (iLynette) [inline item.rb]: By default, @label will be nil so you can just leave it as `@label`

File: item.rb (hunk 1 above)
Comment (daudi13) [reply item.rb]: Thanks for pointing that out @iLynette

File: item.rb (hunk 2 above)
Comment (daudi13) [reply item.rb]: Just fixed it. :+1:

**Review by iLynette — APPROVED** [review]
Hello 👋 ,

APPROVED 🟢🟢✔
Great job making it this far.

You have implemented can_be_archived?() and move_to_archive()
No linter errors
Descriptive commit message
You are free to merge the changes 🐱‍🏍🎉

Check the comments section for optional suggestions

**Pull request merged**